calico/common.py

```diff
@@ -24,9 +24,17 @@
 import logging
 import logging.handlers
 import os
+import re
 import sys
 import errno
 
+# Various useful regular expressions.
+IPV4_REGEX = re.compile("^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$")
+IPV6_REGEX = re.compile("^[a-f0-9]+:[:a-f0-9]+$")
```

Comment:
I have replaced these with utility functions, but kept the regexes within the functions. I think that is good enough for now - we could make it better, but it feels like premature optimisation to load a library or write any more complex code to get a 0.0001% use case.